changelog

```diff
@@ -1,5 +1,11 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 0.2.1
+----------------------------
+- Picks up common@0.0.6
```

Comment:
Shouldn't this be `common@0.0.7`?

Reply:
Yes, it should!

Reply:
Fixed in 94c9463538127946179308626e8452dd20abe50d